modules/apps/object/object-service/src/main/java/com/liferay/object/internal/field/business/type/RichTextObjectFieldBusinessType.java

```diff
@@ -67,6 +69,16 @@ public String getName() {
 		return ObjectFieldConstants.BUSINESS_TYPE_RICH_TEXT;
 	}
 
+	@Override
+	public Map<String, Object> getProperties(
+		ObjectField objectField,
+		ObjectFieldRenderingContext objectFieldRenderingContext) {
+
+		return HashMapBuilder.<String, Object>put(
+			"objectFieldLocalized", objectField.isLocalized()
```

Comment:
Hey @pedro-oliveira446,
I searched the words "objectFieldLocalized" and "localizedObjectFields" in the object's modules. I saw that the option "localizedObjectFields" is the one used for variables. Also, it is better to finish the variable with the entity's name like "xyzObjectField".
Do you agree? Could you replace it please?